src/test/resources/generate-mojo-full-pom.xml

```diff
@@ -35,6 +35,9 @@
                                 <url>https://license</url>
                                 <name>MIT</name>
                             </license>
+                            <extensions>
+                                <x-custom-property>My custom property</x-custom-property>
```

Comment:
I am wondering whether the integration test could verify that this extension is actually being output?

Reply:
No problem - the current setup is pretty much just a basic verification it could be nice to enhance it. Only thing is to make sure the test is still reasonably robust. I see your point in the pretty printing, however, I think I would personally opt for parsing the yaml/json and validating the relevant properties instead of doing a strict check of the whole output. I could worry that an upgrade of, e.g., Swagger that, for instance, just changed the order of properties in the output would break the test even there is no semantic difference.

Reply:
Sorry for the delay, was away from keyboard during the Christmas break. I now completely changed the unit tests by adding some expected generated Swagger outputs. Each test checks that the generated Swagger specs, when parsed again as an `OpenAPI` object, matches the equivalent expected object. I checked that `OpenAPI.equals()` performs a deep copy of all the properties, changing a single character in for example one of the expected outputs makes the tests fail.

Reply:
No problem. I will take a look during this week.